components/browser/tabstray/src/main/java/mozilla/components/browser/tabstray/TabsAdapter.kt

```diff
@@ -65,4 +86,20 @@ open class TabsAdapter(
     override fun onTabsMoved(fromPosition: Int, toPosition: Int) = notifyItemMoved(fromPosition, toPosition)
 
     override fun onTabsChanged(position: Int, count: Int) = notifyItemRangeChanged(position, count)
+
+    companion object {
+        /**
+         * Payload used in onBindViewHolder for a partial update of the current view.
+         *
+         * Signals that the currently selected tab should be highlighted. This is the default behavior.
+         */
+        const val PAYLOAD_HIGHLIGHT_SELECTED_ITEM = "Should highlight the current tab"
```

Comment:
Should these just be integers? That could suffice with indicating the type, maybe a resource generated value in the `ids.xml` so we don't have to keep count of integer increments.

```suggestion
        const val PAYLOAD_HIGHLIGHT_SELECTED_ITEM = R.id.payload_highlight_selected_item
```

Reply:
Sure! Thanks!